pkg/vsphere/datastore/datastore.go

```diff
@@ -239,7 +239,7 @@ func (d *Helper) Mv(ctx context.Context, fromPath, toPath string) error {
 	from := path.Join(d.RootURL, fromPath)
 	to := path.Join(d.RootURL, toPath)
 	log.Infof("Moving %s to %s", from, to)
-	err := tasks.Wait(ctx, func(context.Context) (tasks.Waiter, error) {
+	err := tasks.Wait(ctx, func(context.Context) (tasks.ResultWaiter, error) {
```

Comment:
Why did you make this change? If you are going to change this, might as well use `object.Task` from govmomi.